meta.yaml

```diff
@@ -0,0 +1,134 @@
+{% set name = "flytekit" %}
+{% set version = "0.24.0b0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 21c1560de9a3af594ac542f31d6888f5232540f5338f277493c31d4e49c713a1
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - pyflyte-execute=flytekit.bin.entrypoint:execute_task_cmd
+    - pyflyte-fast-execute=flytekit.bin.entrypoint:fast_execute_task_cmd
+    - pyflyte-map-execute=flytekit.bin.entrypoint:map_execute_task_cmd
+    - pyflyte=flytekit.clis.sdk_in_container.pyflyte:main
+    - flyte-cli=flytekit.clis.flyte_cli.main:_flyte_cli
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - checksumdir >=1.2.0
+    - click >=6.6,<8.0
+    - cloudpickle >=2.0.0
+    - croniter >=0.3.20,<4.0.0
+    - dataclasses-json >=0.5.2
+    - deprecated >=1.0,<2.0
+    - diskcache >=5.2.1
+    - docker-image-py >=0.1.10
+    - docstring-parser >=0.9.0
+    - flyteidl >=0.21.4
+    - grpcio >=1.3.0,<2.0
+    - keyring >=18.0.1
+    - marshmallow-jsonschema >=0.12.0
+    - natsort >=7.0.1
+    - pandas >=1.0.0,<2.0.0
+    - protobuf >=3.6.1,<4
+    - pyarrow >=2.0.0,<4.0.0
+    - python >=3.6
+    - python-dateutil <=2.8.1,>=2.1
+    - python-json-logger >=2.0.0
+    - pytimeparse >=1.1.8,<2.0.0
+    - pytz >=2017.2,<2018.5
+    - requests >=2.18.4,<3.0.0
+    - responses >=0.10.7
+    - retry ==0.9.2
+    - singledispatchmethod   # [ py < 38 ]
+    - six >=1.9.0,<2.0.0
+    - sortedcontainers >=1.5.9<3.0.0
+    - statsd >=3.0.0,<4.0.0
+    - urllib3 >=1.22,<2.0.0
+    - wheel >=0.30.0,<1.0.0
+    - wrapt >=1.0.0,<2.0.0
+
+test:
+  imports:
+    - flytekit
+    - flytekit.bin
+    - flytekit.clients
+    - flytekit.clis
+    - flytekit.clis.auth
+    - flytekit.clis.flyte_cli
+    - flytekit.clis.sdk_in_container
+    - flytekit.common
+    - flytekit.common.core
+    - flytekit.common.exceptions
+    - flytekit.common.mixins
+    - flytekit.common.tasks
+    - flytekit.common.tasks.sagemaker
+    - flytekit.common.types
+    - flytekit.common.types.impl
+    - flytekit.configuration
+    - flytekit.contrib
```

Comment:
actually we can drop some of the packages. Refer to - https://docs.flyte.org/projects/flytekit/en/latest/contributing.html#quick-background

Reply:
I will edit this section.